lib/Polaris.Class.ps1

```diff
@@ -292,8 +296,28 @@ class Polaris {
             $this.Listener.Prefixes.Add("$($ListenerPrefix)://localhost:" + $this.Port + "/")
         }
 
+        switch ( $Auth ) {
+            Basic {
+                $this.Listener.AuthenticationSchemes = "Basic"
+            }
+            Negotiate {
+                $this.Listener.AuthenticationSchemes = "Negotiate"
+            }
+            NTLM {
+                $this.Listener.AuthenticationSchemes = "NTLM"
+            }
+            Anonymous {
+                $this.Listener.AuthenticationSchemes = "Anonymous"
+            }
+            default {
+                $this.Listener.AuthenticationSchemes = "Anonymous"
+            }
+        }
```

Comment:
I think this is duplicate to what you are already doing with the validateset attribute and defaulting to Anonymous. You can remove the entire switch statement and replace it with:

```ps
$this.Listener.AuthenticationSchemes = $Auth
```

You'd still be considered safe because the user can only enter a choice that's in your validate set and if they don't use any choice at all it will choose anonymous for them.